frontend/src/layout/navigation-3000/components/NavbarButton.tsx

```diff
@@ -53,25 +53,18 @@ export const NavbarButton: FunctionComponent<NavbarButtonProps> = React.forwardR
         if (!isNavCollapsedActually) {
             content = shortTitle || title
             if (tag) {
-                if (tag === 'alpha') {
-                    content = (
-                        <>
-                            <span className="grow">{content}</span>
-                            <LemonTag type="completion" size="small" className="ml-2">
-                                ALPHA
-                            </LemonTag>
-                        </>
-                    )
-                } else if (tag === 'beta') {
-                    content = (
-                        <>
-                            <span className="grow">{content}</span>
-                            <LemonTag type="warning" size="small" className="ml-2">
-                                BETA
-                            </LemonTag>
-                        </>
-                    )
-                }
+                content = (
+                    <>
+                        <span className="grow">{content}</span>
+                        <LemonTag
+                            type={tag === 'alpha' ? 'completion' : tag === 'beta' ? 'warning' : 'warning'}
```

Comment:
```suggestion
                            type={tag === 'alpha' ? 'completion' : 'warning'}
```

Reply:
Let's use success like in the old UI
```suggestion
                            type={tag === 'alpha' ? 'completion' : tag === 'beta' ? 'warning' : 'success'}
```